File: ApiBooksServer/Database/BooksTable/BooksTable.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ApiBooksServer.Database.BooksTable;

public class BooksTable
{
    [Key]
    public int BookId { get; set; }
    public string BookName { get; set; }
    public int YearCreate { get; set; }
    [ForeignKey("GenreTable")]
    public int GenreId  { get; set; };
    public int SaledBooks { get; set; }
    public byte[] ImageBooks { get; set; }
    public string Description { get; set; }
    public virtual GenreTable.GenreTable Genre { get; set; }
}

File: src/Database/model/BooksTable.py
from datetime import date

from sqlalchemy import Integer , ForeignKey
from sqlalchemy.orm import Mapped , mapped_column

from Database.connect.database_connect import Base


# Таблица книг
class BooksTable ( Base ) :
	__tablename__ = "books_Table"
	book_id: Mapped [ int ] = mapped_column ( Integer , primary_key = True , autoincrement = True )
	book_name: Mapped [ str ] = mapped_column ( nullable = False )
	year_Create: Mapped [ date ] = mapped_column ( nullable = False )
	genre_Id: Mapped [ int ] = mapped_column ( ForeignKey ( 'genre_table.genre_id' , ondelete = 'CASCADE' ) )
	saled_books: Mapped [ int ] = mapped_column ( nullable = False )
	image_books: Mapped [ bytes ] = mapped_column ( nullable = False )
	description: Mapped [ str ] = mapped_column ( nullable = False )
